app/controllers/dashboard_controller.rb

```diff
@@ -1,5 +1,9 @@
 class DashboardController < ApplicationController
   before_action :authenticate
   def show
+    geolocation = GoogleGeolocationService.new
+    current_user.lat = geolocation.coordinates[:location][:lat]
```

Comment:
Uses the user's current lat and long that refreshes each time they visit the dashboard show page.